core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -594,6 +598,21 @@ public boolean isObjectStorage() {
     return true;
   }
 
+  /**
+   * Checks if the root of the ObjectUnderFileSystem is listable. This is done once and cached
+   * per UFS to reduce time spent checking if the root is accessible.
+   *
+   * @return whether the root of the UFS is listable and return false if any exception is thrown
+   */
+  public boolean isRootListable() {
+    try {
+      return getObjectListingChunkForPath("", true) != null;
+    } catch (Exception e) {
+      LOG.debug("Unable to list root of bucket {}:", super.mUri.toString(), e.getMessage());
```

Comment:
Is `e.getMessage()` used in the message? And, do we want the full stack trace of the exception, or do we just want the message?

Reply:
Good catch. There is no message for this and we do want the full exception.